.github/workflows/update_docs.yml

```diff
@@ -10,6 +10,8 @@ on:
     secrets:
       REPO_SCOPED_TOKEN:
         required: true
+    permissions:
```

Comment:
out of curiosity when did this stop working?

Reply:
it didn't work for my pr but it worked for a dummy pr that Yen created. Yen said that I should manually updated the doc in my pr and investigate later

Reply:
do we know why it worked for Yen's dummy PR but not for your PR?

Reply:
might be worth making an item for this

Reply:
We never figured out why it failed on my PR on particular. After making some changes, it worked again.

Reply:
Ok I'll close this for now but won't delete the branch!